### PR TITLE
Create OVON Interoperability Precis.md

### PR DESCRIPTION
**Pull request by jcstine**
The Precis of the interoperability initiative of the Open Voice Network

### CODE REVIEW

File: folderAWG/OVON-Interoperability-Precis.md [hunk 1]
```diff
@@ -0,0 +1,189 @@
...
+
+This overview of the Open Voice Network’s approach to voice agent interoperability is intended to provide a lay-level introduction for prospective participants in and sponsors of this important work. 
+
+Documentation of the Open Voice Network (OVON) Interoperability Initiative will be stored in the Open Voice Network’s GitHub repository, which can be accessed here:  github.com/open-voice-network/docs/folderAWG/OVON-Interoperability-Precis.md
+
+### Working Definition of Interoperability
+
+**The ability for voice agents of different platforms or technological parentage to share dialogs, data, context, and controls.**
+
+### III.	The Voice-Expanded Web: The World for Which We Build
```
Comment (JohnTrammellTarget) [inline folderAWG/OVON-Interoperability-Precis.md:11]: We should start with section 1

File: folderAWG/OVON-Interoperability-Precis.md [hunk 2]
```diff
@@ -0,0 +1,189 @@
...
+
+* review business data;
+* adjust ambient lighting;
+* guide transportation systems;
+* explore virtual worlds with fictional characters and synthetic voices; and, 
+* manage their personal assistant, and share personal data through voice-based conversation and consent.
+
+Should this transition be supported and accelerated by broadly adopted open technology standards, significant growth can be predicted in both the size and breadth of the market. Value will be realized through the ability to connect, to create layers of ever-more valuable connection, and the acquisition and deep analysis of data. Agent-to-agent interoperability addresses not just an issue of technology, but the economic question that looms before the voice-expanded World Wide Web: who will reap the value?  
+
+### IV.	OPEN VOICE NETWORK 1.0 HYPOTHESES FOR AGENT-TO-AGENT INTEROPERABILITY
```
Comment (pmotch) [inline folderAWG/OVON-Interoperability-Precis.md:29]: Remove extra spaces between numeral and title. Several sections like this.

File: folderAWG/OVON-Interoperability-Precis.md [hunk 3]
```diff
@@ -0,0 +1,189 @@
+## VOICE AGENT INTEROPERABILITY
+
+This overview of the Open Voice Network’s approach to voice agent interoperability is intended to provide a lay-level introduction for prospective participants in and sponsors of this important work. 
+
+Documentation of the Open Voice Network (OVON) Interoperability Initiative will be stored in the Open Voice Network’s GitHub repository, which can be accessed here:  github.com/open-voice-network/docs/folderAWG/OVON-Interoperability-Precis.md
+
+### Working Definition of Interoperability
```
Comment (pmotch) [inline folderAWG/OVON-Interoperability-Precis.md:7]: Consider consistency across header styles. Some are all caps, some are Chicago Style, some are lower case.

File: folderAWG/OVON-Interoperability-Precis.md [hunk 4]
```diff
@@ -0,0 +1,189 @@
...
+
+This infrastructure will include:
+
+* A set of well-defined messages for starting, stopping, suspending, and resuming voice agents;
+* Protocols consisting of sequences of messages that implement the enhancement; 
+* A set of self-defined data formats for transferring data among agents. 
+
+The six enhancements are as follows:  
+
+1.	 Remote invocation of a second voice agent.  To locate and invoke a second voice agent, regardless of platform.
```
Comment (pmotch) [inline folderAWG/OVON-Interoperability-Precis.md:105]: Consider using markup for ordered bulleted lists and remove extra spaces.

File: folderAWG/OVON-Interoperability-Precis.md [hunk 5]
```diff
@@ -0,0 +1,189 @@
...
+We seek to develop standards that will enable conversational agents of different platforms (open, proprietary, hybrid) to share dialogs, data, context, and controls. 
+
+Q: Will developers be able to insert Open Voice Network enhancements directly into their products?
+
+A: No. Enhancements will likely need to be refined and tailored to be effective in each vendor product as they use different platforms, languages, and implementation strategies. The true value of Open Voice Network interoperability work will most likely be found in the standard message formats, protocols, and data formats, and the algorithms used to implement each enhancement.  
+
+
+### IX.	ABOUT THE OPEN VOICE NETWORK (www.openvoicenetwork.org) 
+
+Vision
```
Comment (pmotch) [inline folderAWG/OVON-Interoperability-Precis.md:157]: A few of the sub headers in this section (e.g. "Vision", "Mission", etc) could use markup to make them easier to read. Banal thing.

**Pull request closed without merging**